doc/lsst.meas.algorithms/tasks/lsst.meas.algorithms.SourceDetectionTask.rst

```diff
@@ -0,0 +1,108 @@
+.. lsst-task-topic:: lsst.meas.algorithms.SourceDetectionTask
+
+###################
+SourceDetectionTask
+###################
+
+``SourceDetectionTask`` Detect positive and negative sources on an exposure
+and return a new table.SourceCatalog
+
+.. _lsst.meas.algorithms.SourceDetectionTask-api:
+
+Python API summary
+==================
+
+.. lsst-task-api-summary:: lsst.meas.algorithms.SourceDetectionTask
+
+.. _lsst.meas.algorithms.SourceDetectionTask-subtasks:
+
+Retargetable subtasks
+=====================
+
+.. lsst-task-config-subtasks:: lsst.meas.algorithms.SourceDetectionTask
+
+.. _lsst.meas.algorithms.SourceDetectionTask-configs:
+
+Configuration fields
+====================
+
+.. lsst-task-config-fields:: lsst.meas.algorithms.SourceDetectionTask
+
+.. _lsst.meas.algorithms.SourceDetectionTask-examples:
+
+Examples
+========
+
+This code is in `measAlgTasks.py` in the examples directory, and can be run as *e.g.*
+
+.. code-block:: py
+
+   examples/measAlgTasks.py --doDisplay
+
+The example also runs the SingleFrameMeasurementTask; see meas_algorithms_measurement_Example
+for more explanation.
+
+Import the task (there are some other standard imports; read the file if you're confused)
+
+.. code-block:: py
+
+   from lsst.meas.algorithms.detection import SourceDetectionTask
+
+We need to create our task before processing any data as the task constructor can add an
+extra column to the schema, but first we need an almost-empty Schema
+
+.. code-block:: py
+
+   schema = afwTable.SourceTable.makeMinimalSchema()
+
+after which we can call the constructor:
+
+.. code-block:: py
+
+  config = SourceDetectionTask.ConfigClass()
+  config.thresholdPolarity = "both"
+  config.background.isNanSafe = True
+  config.thresholdValue = 3
+  detectionTask = SourceDetectionTask(config=config, schema=schema)
+
+We're now ready to process the data (we could loop over multiple exposure/catalogues using
+the same task objects). First create the output table:
+
+.. code-block:: py
+
+   tab = afwTable.SourceTable.make(schema)
```

Comment:
Might as well tidy this up and replace the ambiguous "tab" with "table" or "catalog" here, and just below.